client/src/Q&As/Modal.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react';
+import styled, {css} from 'styled-components';
+
+const Modal = ({title, subTitle }) => {
+  const [show, setShow] = useState(false);
+
+  const showModal = (e) => {
+    setShow(true);
+  };
+
+  /*
+  <div><button onClick={(e) => { showModal(); }}>Add a question</button></div>
+  */
+
+  return (
+    <div className="qanda-modal">
+      <div className="qanda-modal-content">
+        <div className="qanda-modal-header">
+          <h4>{title}</h4>
+          <h2>{subTitle}</h2>
+        </div>
+        <div className="qanda-modal-body">
+          This is the body, form goes in here
+        </div>
+        <div className="qanda-modal-footer">
+          <button>Submit</button>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+
+
+export default Modal;
```

Comment:
Minor thing I noticed this week that I also forgot about until now: every file is supposed to end with a new/blank line. Otherwise we get a little red indicator at the end of the file mentioning it wasn't ended with a new line.

Reply:
I didn't know that, thank you 👏